documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -110,28 +110,29 @@ spec:
           name: <source_network_name>
         destination:
           type: pod
-          name: pod
+          name: pod <5>
           namespace: {namespace}
-    datastores: <5>
-      - source: <6>
-          id: <source_datastore_mor> <7>
+    datastores: <6>
+      - source: <7>
+          id: <source_datastore_mor> <8>
           name: <source_datastore_name>
         destination:
           storageClass: standard
-  vms: <8>
-    - id: <source_vm_mor> <9>
+  vms: <9>
+    - id: <source_vm_mor> <10>
     - name: <source_vm_name>
 EOF
 ----
 <1> Specify the name of the `Plan` CR.
 <2> You can create multiple network mappings for source and destination networks.
 <3> You can use either the `id` _or_ the `name` parameter to specify the source network.
 <4> _Managed object reference_ of the source network.
-<5> You can create multiple storage mappings for source data stores and destination storage classes.
-<6> You can use either the `id` _or_ the `name` parameter to specify the source data store.
-<7> _Managed object reference_ of the source data store.
-<8>  You can use either the `id` _or_ the `name` parameter to specify the source VM.
-<9> _Managed object reference_ of the source VM.
+<5> Specify the name of the default migration transfer network. The default is the `pod` network, which is not optimal for disk transfer.
```

Comment:
Here, we're configuring the network mapping, which has nothing to do with the migration network. I wouldn't change this.

@mansam what attribute is set to specify the migration network?

Reply:
OK. I will leave this comment unresolved but change the text.

Reply:
@mansam Still waiting for your input.
@fdupont-redhat Do you have any input for selecting OCP network from the CLI?